lib/Minz/Session.php

```diff
@@ -66,9 +66,11 @@ public static function unset_session($force = false) {
 	 */
 	public static function keepCookie($l) {
 		// Get the script_name (e.g. /p/i/index.php) and keep only the path.
-		$cookie_dir = empty($_SERVER['SCRIPT_NAME']) ? '' : $_SERVER['SCRIPT_NAME'];
-		$cookie_dir = dirname($cookie_dir);
-		session_set_cookie_params($l, $cookie_dir, '', false, true);
+		$cookie_dir = empty($_SERVER['REQUEST_URI']) ? '/' : $_SERVER['REQUEST_URI'];
+		if (substr($cookie_dir, -1) !== '/') {
+			$cookie_dir = dirname($cookie_dir) . '/';
+		}
+		session_set_cookie_params($l, $cookie_dir, '', false, false);
```

Comment:
It was a mistake. Corrected https://github.com/Alkarex/FreshRSS/commit/189e790f32d4e389cf1dc6da669a579717fff436